README.md

```diff
@@ -1,96 +1,154 @@
+
 ## Contributing to Swift Numerics
-Swift Numerics is a standalone library separate from the core Swift project.
-In practice, it will act as a staging ground for some APIs that may eventually be incorporated into the Swift Standard Library, and when that happens such changes will be proposed to the Swift Standard Library using the established evolution process of the Swift project.
 
-Swift Numerics uses GitHub issues to track bugs and features. We use pull requests for development.
+Swift Numerics is a standalone library that is separate from the core Swift
+project. In practice, it will act as a staging ground for some APIs that may
+eventually be incorporated into the Swift Standard Library. When that happens,
+such changes will be proposed to the Swift Standard Library using the
+established evolution process of the Swift project.
+
+Swift Numerics uses GitHub issues to track bugs and features. We use pull
+requests for development.
+
+### How to propose a new module
 
-To propose a new module:
 1. Raise an issue with the [new module] tag.
 2. Raise a PR with an implementation sketch.
-3. Once you have some consensus, ask an admin to create a feature branch against which PRs can be raised.
-4. When the design has stabilized and is functional enough to be useful, raise a PR to merge the new module to master.
+3. Once you have some consensus, ask an admin to create a feature branch against
+   which PRs can be raised.
+4. When the design has stabilized and is functional enough to be useful, raise a
+   PR to merge the new module to master.
+
+### How to propose a new feature for an existing module
 
-To propose a new feature for an existing module:
 1. Raise an issue with the [enhancement] tag.
 2. Raise a PR with your implementation, and discuss the implementation there.
-3. Once there is a consensus that the new feature is desirable and the design is suitable, it can be merged.
+3. Once there is a consensus that the new feature is desirable and the design is
+   suitable, it can be merged.
 
-To fix a bug, or make smaller improvements:
-1. Raise a PR with your change. Be sure to add test coverage for whatever changes you are making.
+### How to fix a bug, or make smaller improvements
 
-Questions about how to use Swift Numerics modules, or issues that are not clearly bugs can be discussed in the ["Swift Numerics" section of the Swift forums.](https://forums.swift.org/c/related-projects/swift-numerics)
+1. Raise a PR with your change. 
+2. Make sure to add test coverage for whatever changes you are making.
+
+### Forums
+
+Questions about how to use Swift Numerics modules, or issues that are not
+clearly bugs can be discussed in the ["Swift Numerics" section of the Swift
+forums](https://forums.swift.org/c/related-projects/swift-numerics).
 
 ## Modules
-1. [RealModule](Sources/RealModule/README.md)
-2. [ComplexModule](Sources/ComplexModule/README.md)
+
+1. About [`RealModule`](Sources/RealModule/README.md)
```

Comment:
I don't think "About" adds any clarity here (it actually confuses it). This is a list of modules, and the module is not "About RealModule" it is "RealModule".

Reply:
@stephentyrone Sounds good! Maybe we can have the title of the section named "# More on modules" or something along those lines? What each link is about/does can be quite important to increase clarity, better a11y, etc. WDYT?

Reply:
I think the heading is fine as-is. It is a list of modules, not a section “on modules.”

Reply:
I am just trying to follow commonly accepted guidelines of tech writing for good docs. 

The idea in our case is to help any user - whether they're new to Swift or Swift Numerics - to understand whether they should click on each link (this is part of the user experience/human-computer interface fields). So, in a way, this contributes to inclusiveness and accessibility, which are important to keep in mind when writing doc specs.

A few words with basic info can save readers a trip to an external site that may not want to go to (as per the Google style guide, as I couldn't find any relevant info on Microsoft or Apple's guides).

- For instance, this shows it's a README and I'd probably click on it, since READMEs are mostly common knowledge:

```
[`RealModule` - README](Sources/RealModule/README.md)
```

```
[About `RealModule`](Sources/RealModule/README.md)
```

- Compare this the original, where it's not too clear whether the user, who hasn't spent as much time with Swift Numerics as us, should click on the link. (Is it an API doc, a `.swift` file, a blog post, or an official Apple reference?)

```
[`RealModule`](Sources/RealModule/README.md)
```

Anyway, these are just suggestions. Maybe Apple's tech writers can help here.

Cheers!